src/world.rs

```diff
@@ -1,7 +1,7 @@
 use crate::config::Config;
 use crate::envelope::Protocol;
 use crate::ip::IpVersionAddrIter;
-use crate::{config, Dns, Host, ToIpAddr, ToIpAddrs, Topology, TRACING_TARGET};
+use crate::{config, for_pairs, Dns, Host, ToIpAddr, ToIpAddrs, Topology, TRACING_TARGET};
```

Comment:
Let's also move `for_pairs` into here since I believe that is the only place we use it now.

Reply:
It is also used in `sim.rs` should it still be moved to `world.rs`?

Reply:
Ok, pub(crate) on the lib seems fine.